content/en/docs/gke/deploy/deploy-cli.md

```diff
@@ -112,19 +112,25 @@ kubeflow is defined. These are described in the table below.
 kpt setter | Description |
 -----------|-------------|
 mgmt-ctxt | This is the name of the KUBECONFIG context for the management cluster; this kubecontext will be used to create CNRM resources for your Kubeflow deployment. **The context must set the namespace to the namespace in your CNRM cluster where you are creating CNRM resources for the managed project.**|
-gcloud.core.project| The project you want to deploy in |
+name| This is the name of the Kubeflow deployment |
 location | The zone or region you want to deploy in |
-gcloud.compute.region | The region you are deploying in |
 gcloud.compute.zone | The zone to use for zonal resources; must be in gcloud.compute.region |
+gcloud.core.project| The project you want to deploy in |
+email| The email adderss of the admin |
```

Comment:
Hey @subodh101 Can you check the spelling of "address" here pls? Thanks!

Reply:
To improve formatting, can you also check that you have a blank line added before and after each code/command-line block please? Thank you!

Reply:
Thanks!

Reply:
@subodh101 I noticed small thing - can you please change `bash` to `shell` in code blocks? I'm not sure if it makes a huge difference in terms of rendering on the site, but it's probably more proper. Thanks 👍 

```
    ```shell
    ...
    ```
```